include/HubFramework/HUBContentOperationContext.h

```diff
@@ -0,0 +1,55 @@
+/*
+ *  Copyright (c) 2016 Spotify AB.
+ *
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ *  or more contributor license agreements.  See the NOTICE file
+ *  distributed with this work for additional information
+ *  regarding copyright ownership.  The ASF licenses this file
+ *  to you under the Apache License, Version 2.0 (the
+ *  "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+#import "HUBConnectivityState.h"
+
+@protocol HUBFeatureInfo;
+@protocol HUBViewModelBuilder;
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ *  Protocol defining the public API of a content operation context
+ *
+ *  This API is currently only used with `HUBBlockContentOperation`. For more information
```

Comment:
Yeah, I wrote this in the PR description 😉 

> This contextual object API is currently only used for the newly introduced block operations, but the plan is to migrate all HUBContentOperations to using it, instead of having multiple parameters as input.